rust/private/rustc.bzl

```diff
@@ -970,6 +970,8 @@ def construct_arguments(
     if hasattr(ctx.attr, "_extra_exec_rustc_flag") and is_exec_configuration(ctx):
         rustc_flags.add_all(ctx.attr._extra_exec_rustc_flag[ExtraExecRustcFlagsInfo].extra_exec_rustc_flags)
 
+    rustc_flags.add_all(toolchain.rustc_flags)
```

Comment:
I'd like to have 2 toolchain-specific fields, distinguishing between target-specific and exec-specific flags, similarly to _extra_{exec_}_rustc_flags above (say to be able to define a sanitizer-enabled rust_toolchain that uses `-Zsanitizer=address` in target configuration and not in exec configuration). For consistency, let's call them `extra_rustc_flags` and `extra_exec_rustc_flags`. Also let's move them before line 960 so that the user-supplied extra flags appear after the toolchain-supplied ones.
Also it would be nice to come up with a test for these.